src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageStatus.cs

```diff
@@ -1,15 +1,15 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 namespace NuGet.PackageManagement.UI
 {
     public enum PackageStatus
     {
-        NotInstalled,
+        NotInstalled = 0,
```

Comment:
Why do we need this change?

Reply:
I changed view model to no longer set the field to notinstalled. i'm used to setting the default item in an enum with 0. i might be able to live by reverting both. 
thoughts?

Reply:
It defaults to 0, so it doesn't add much. 
I don't care too much in particular though.

Reply:
Feel free to resolve the discussion as you see fit, no preference from my end :)

Reply:
will remove zero, keep other code change cause it is unnecessary.